### PR TITLE
Updated with IBM Developer Day

### PR DESCRIPTION
**Pull request by RiyaMRoy04**
Updated with IBM Developer Day Data Science and Artificial Intelligence track

**Details of the conference**:

- Website: https:ibm.biz/ibmdevday
- Date:
- Venue:
- Description:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ To add a conference to the list, please [contribute](#contributing)!
 | [Deccan RubyConf 2018](https://www.deccanrubyconf.org/) | 4 Aug | Sheraton Grand Hotel, Pune | Pune extension of RubyConf India organized by same folks organizing RubyConf India |
 | [DroidJam](https://www.droidjam.in) | 13-14 July | Sheraton Grand, Bangalore | Conference on Android |
 | [Selenium Conf 2018](https://2018.seleniumconf.in/) | 28-30 June | Hotel Lalit Ashok, Bangalore  | Conference on Selenium Web Browser Automation tool |
+| [IBM Developer Day](https://ibm.biz/ibmdevday) | 30 June | 1st Floor,D2 Block,IBM India Pvt Ltd,Manyata Tech Park,Bangalore  |  A day long event designed for developers to enhance their skills and gain hands on experience with the latest technology in areas of Artificial Intelligence.|
```
Comment (nikhita) [inline README.md]: In the first column, please change it to `IBM Developer Day (AI)`.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ To add a conference to the list, please [contribute](#contributing)!
...
+| [IBM Developer Day](https://ibm.biz/ibmdevday) | 26 May | 1st Floor,D2 Block,IBM India Pvt Ltd,Manyata Tech Park,Bangalore  |  A day long event designed for developers to enhance their skills and gain hands on experience with the latest technology in areas of Data Science.|
```
Comment (nikhita) [inline README.md]: In the first column, please change it to `IBM Developer Day (Data Science)`.

File: README.md (hunk 1 above)
Comment (nikhita) [reply README.md]: For both, in the venue - just `Manyata Tech Park, Bangalore` is fine. It is short and concise this way. :)

**Comment by RiyaMRoy04** [comment]
@nikhita Updated with the changes! 👍

**Pull request merged**